openslides/agenda/static/js/agenda/base.js

```diff
@@ -66,6 +75,27 @@ angular.module('OpenSlidesApp.agenda', ['OpenSlidesApp.users'])
                                element.id == self.content_object.id;
                     };
                     return typeof _.findKey(projector.elements, predicate) === 'string';
+                },
+                // project list of speakers
+                projectListOfSpeakers: function() {
+                    return $http.post(
+                        '/rest/core/projector/1/prune_elements/',
+                        [{name: 'agenda/item', id: this.id, list_of_speakers: true}]
```

Comment:
Never mind what we should write in the db.

But the template code should be in the agenda app. In other case we would have to add templates in any app that can be a related item.

@normanjaeckel you should choose what is the easiest way on the server side.